### DISCUSSION TITLE
Einstein for Developers: Upcoming Open Beta Announcement

### DISCUSSION THREAD

**Discussion by AnanyaJha** [forcedotcom/Einstein-GPT-for-Developers#11] [Announcements]
Thank you all for your participation in the Einstein for Developers Pilot Program - your feedback has helped our team understand what you and your developers are looking for from AI-based developer tooling and helped us drive the product forward. **Einstein for Developers will be moving into an Open Beta stage this Friday, September 8th.**

Some of you will be seeing the transition from Pilot to Open Beta throughout this week - **Open Beta will be self-enabled** so you'll need to ask a System Administrator to read through the new Terms & Conditions before enabling Einstein for Developers in each of your development orgs. I'll be sharing updated documentation and guidance for the Open Beta setup and installation experience with you and your team. 

This Friday, you'll also see this repository be open sourced to encourage collaboration with our wider Open Beta audience - so stay tuned for the updates!

We're so thankful for your feedback during the Pilot period and we're looking forward to working closely with you during Open Beta as well! Let us know if you have any questions and we'd be happy to help.

**Comment by abrantes23** [comment]
Our team wants to thank you Annaya and Salesforce for this opportunity to learn, develop and help to shape Einstein for Developers! Look forward to see the presentations on DreamForce 2023!
